server.js

```diff
@@ -0,0 +1,86 @@
+const express = require('express')
+const bodyParser = require('body-parser')
+const resource = require('./public/resource.js')
+const app_modules = require('./public/app.js')
+const app = express()
+
+app.use(express.static('public'))
+app.use(bodyParser.json())
+
+let contenuAjout = ""
+
+const html = (contenuAjout) => `
+<!doctype html>
+<html lang="en">
+  <head>
+    <title>FaceYourWilder</title>
+    <!-- Required meta tags -->
+    <meta charset="utf-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
+
+    <!-- Bootstrap CSS -->
+    <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/4.0.0/css/bootstrap.min.css" integrity="sha384-Gn5384xqQ1aoWXA+058RXPxPg6fy4IWvTNh0E263XmFcJlSAwiGgFAW/dAiS6JXm" crossorigin="anonymous">
+  </head>
+  <body>
+    
+    <div id='output'>
+      ${contenuAjout}
+    </div>
+
+    <!-- Optional JavaScript -->
+    <!-- jQuery first, then Popper.js, then Bootstrap JS -->
+    <script src="https://code.jquery.com/jquery-3.2.1.slim.min.js" integrity="sha384-KJ3o2DKtIkvYIK3UENzmM7KCkRr/rE9/Qpg6aAZGJwFDMVNA/GpGFF93hXpG5KkN" crossorigin="anonymous"></script>
+    <script src="https://cdnjs.cloudflare.com/ajax/libs/popper.js/1.12.9/umd/popper.min.js" integrity="sha384-ApNbgh9B+Y1QKtv3Rn7W3mgPxhU9K/ScQsAP7hUibX39j7fakFPskvXusvfa0b4Q" crossorigin="anonymous"></script>
+    <script src="https://maxcdn.bootstrapcdn.com/bootstrap/4.0.0/js/bootstrap.min.js" integrity="sha384-JZR6Spejh4U02d8jOt6vLEHfe/JQGiRRSQQxSfFWpi1MquVdAyjUar5+76PVCmYl" crossorigin="anonymous"></script>
+    <script src="contenu.js"></script>
+  </body>
+</html>`
+
+// i will do something when receiving the homepage request
+app.get('/', (req, res) => {
+  const HtmlAjout = `
+  <div>HTML homepage</div>
+  `
+  res.send(html(HtmlAjout+htmlFooter))
+  res.end()
+})
+
```

Comment:
why so many roads ? did you understand what is a SPA ? did you check how does the Pirates exemple work ? Did you plan to reload every time you hit a road ?

Reply:
We have many routes because we plan to create many sections (cf. our user stories). Actually, we're not really sure how do routes work.